[Patch/Rename] fabric-crash-report-info-v1 && fabric-message-api-v1

coredex-source opened the pull request.

coredex-source committed:
fabric-crash-report-info-v1

- Does not seem to need anything else renamed.

Earthcomputer commented:
Hi, can you let us know on Discord or in the main PR when you're planning on working on a module, so that others don't start working on the same module?

coredex-source committed:
fabric-message-api-v1 - commit.1

- ClientReceiveMessageEvents -> ClientLogChatMessageEvents

coredex-source commented:
@Earthcomputer Definitely yeah, my bad.

coredex-source committed:
fabric-message-api-v1 - commit.2

- Patch files previously dependent on ClientReceiveMessageEvents.

coredex-source committed:
fabric-message-api-v1 - commit.3

ClientSendMessageEvents -> ClientHandleChatInputEvents

coredex-source committed:
Fix build

deirn commented on fabric-message-api-v1/src/client/java/net/fabricmc/fabric/api/client/message/v1/ClientHandleChatInputEvents.java:
```diff
@@ -34,9 +34,9 @@ private ClientSendMessageEvents() {
-	public static final Event<AllowChat> ALLOW_CHAT = EventFactory.createArrayBacked(AllowChat.class, listeners -> (message) -> {
-		for (AllowChat listener : listeners) {
-			if (!listener.allowSendChatMessage(message)) {
+	public static final Event<IsChatAllowed> IS_CHAT_ALLOWED = EventFactory.createArrayBacked(IsChatAllowed.class, listeners -> (message) -> {
```
Where does `IS_CHAT_ALLOWED` comes from?

coredex-source replied:
Mojang: Minecraft.ChatStatus.isChatAllowed
Yarn: MinecraftClient.ChatRestriction.allowsChat

coredex-source committed:
fabric-message-api-v1 - commit.4

- Patched ChatListenerMixin.
- Done with client renaming.

coredex-source commented:
Done with renaming the client folder for fabric-message-api-v1.
Changed the title of the PR to make it easier to know what the PR is for.

coredex-source committed:
fabric-message-api-v1 - commit.5 - done.

- ServerMessageEvents -> ServerChatEvents
- ServerMessageDecoratorEvents -> ServerChatDecoratorEvents
- Patched Mixins.
- Patched testmod.
- Finalized renaming of fabric-message-api-v1.

coredex-source committed:
Merge branch 'api-renames' into patch/api-renames
